feature-libs/customer-ticketing/styles/components/_customer-ticketing-list.scss

```diff
@@ -0,0 +1,76 @@
+%cx-customer-ticketing-list {
+  .cx-table {
+    @include media-breakpoint-down(lg) {
+      display: none;
+    }
+
+    thead {
+      border-top: 1px solid var(--cx-color-light);
+
+      th {
+        text-transform: none;
+        font-weight: 600;
+        font-size: 14px;
+        line-height: 20px;
+        color: #000000;
+        text-align: left;
+        padding: 16px 0;
+      }
+    }
+    tbody {
+      tr {
+        td {
+          font-size: 14px;
+          line-height: 20px;
+          color: #000000;
+          padding: 16px 0;
+        }
+      }
+    }
+  }
+
+  .cx-ticketing-list-mobile {
+    @include media-breakpoint-down(lg) {
+      border-top: 1px solid var(--cx-color-light);
+      display: flex;
+    }
+
+    @include media-breakpoint-up(lg) {
```

Comment:
>= 992 to <= 1200, the table is gone 
![Screen Shot 2022-08-31 at 4 53 24 PM](https://user-images.githubusercontent.com/29099466/187780256-f54a8f64-7d78-46e6-b2ba-0300f09bf59b.png)

@morganm58  / @developpeurweb  
This is another point of having a 'different' look and feel table without different styles. Problems like these will occur

Reply:
All in all, I would say the template should be the similar and with similar applied styles to all our listing pages